src/main/fc/config.c

```diff
@@ -177,8 +181,7 @@ uint32_t getGyroLooptime(void) {
     return gyro.targetLooptime;
 }
 
-void validateAndFixConfig(void)
-{
```

Comment:
Why has the format for functions been changed ? Is this some sort of auto format change or deliberate ?

Reply:
Just for consistency.

Reply:
The historic INAV standard has been "brace on new line", and is documented as such via https://github.com/iNavFlight/inav/blob/master/docs/development/Development.md#general-principals (and a link to the basefight coding standard).

So the above is only really consistent with  the other inconsistencies in the source file, if we care about such things.

Reply:
Interesting. It goes back to multiwii. A bit inconsistent between functions and logic, but fair enough. Them's the rules. 

_Curly brace on same line for everything except function definitions_

I'll make it so.